CONTRIBUTING.md

```diff
@@ -63,7 +63,7 @@ Tests will output log output to `./.test-reports/testLog.log` for debugging
 
 #### Common Issues
 
--   Consecutive test runs are currently impacted on VS Code v.1.42.0 due to caching issues. See [VS Code issue #90484](https://github.com/microsoft/vscode/issues/90484). (12 February 2020)
+-   Consecutive test runs were impacted on VS Code **`v1.42.0`** and **`v1.42.1`** due to caching issues. See [VS Code issue #90484](https://github.com/microsoft/vscode/issues/90484). **_This issue has been fixed with the VS Code `v1.43.0` release (9 March 2020)_**
```

Comment:
This is verbose. Minor nit, but I hope to encourage this as a general habit: avoid verbosity. The extra sentence at the end adds no value, just delete it.
```suggestion
-   Consecutive test runs were impacted on VS Code v1.42.x. See [VS Code issue #90484](https://github.com/microsoft/vscode/issues/90484).
```

Reply:
I like having some explicit indication that the fix is no longer required; swapping to past-tense alone is not immediately obvious by itself. I'd personally rather delete the section outright, that way it's a bit less confusing.

Reply:
No objection to deleting the section.

My comment is about verbosity in general. Making things immediately obvious here is not worth the verbosity, because the risk is so low.

Reply:
Opting to remove.